pimpmycause/pimpmycause/templates/registration/create_account.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layout.html" %}
+
+{% load static %}
+
+{% block main %} 
+	<div class="pnl-title page-register">
+		<h1>Register</h1>
+
+        <form method="POST" action="">
+	        {% csrf_token %}
+	        {{ form.as_p }}
```

Comment:
good point i think i can remove that, this wraps form lable and input in a p tag. We don't need that.